bedrock/thunderbird/templates/thunderbird/features.html

```diff
@@ -0,0 +1,243 @@
+          </section>
+        </div>
+      </div>
+    </section><!-- end #tabssearch -->
+
+    <section id="customize-email">
+      <h2>{{ _('Customize Your Email Experience') }}</h2>
+      {% if LANG.startswith('en-') %}
+      <div class="row">
+        <div class="col full">
+          <section>
+            <img class="last" src="{{ static('img/thunderbird/features/filelink.png') }}" alt="">
+            <h3>Large Files Management</h3>
+            <p>Share large files with Thunderbird Filelink!</p>
+            <p>You can now speed up the transfer of large documents by uploading them to an online storage provider and sharing the link instead of sending the file directly as a message attachment. Improve the speed of sending email and avoid message rejection if the recipient's server disallows large files. As an added bonus, you'll also save space in your sent folder and the recipient's inbox.</p>
+          </section>
+        </div>
+      </div>
+      {% endif %}
+      <div class="row">
+        <div class="col">
+          <section>
+            <h3>{{ _('Thunderbird Look &amp; Feel') }}</h3>
+            <p>{{ _('With Personas, lightweight "skins" allow you to change the look and feel of Thunderbird in an instant. Hundreds of skins are available from the latest movies, famous landmarks, and Japanese tattoos. You can also choose from several Themes that dress up all the different icons in Thunderbird.') }}</p>
+          </section>
+          <section>
+            <h3>{{ _('Smart Folders') }}</h3>
+            <p>{{ _('Smart Folders help you manage multiple email accounts by combining special folders like your Inbox, Sent, or Archive folder. Instead of going to the Inbox for each of your mail accounts, you can see all of your incoming email in one Inbox folder.') }}</p>
+          </section>
+        </div>
+        <div class="col">
+          <section>
+            <h3>{{ _('Add-ons Manager') }}</h3>
+            <p>{{ _('Find and install add-ons directly in Thunderbird. You no longer need to visit the add-ons Web site - instead simply fire up the Add-ons Manager. Not sure which add-on is right for you? Ratings, recommendations, descriptions and pictures of the add-ons in action help you make your selection.') }}</p>
+            <img src="{{ static('img/thunderbird/features/addon-manager.png') }}" alt="">
+          </section>
+        </div>
+      </div>
+    </section><!-- end #customize-email -->
+
+    <section id="secureprotect">
+      <h2>{{ _('Secure and Protect Your Email') }}</h2>
+      <div class="row">
+        <div class="col">
+          <section>
+            {% if LANG.startswith('en-') %}
+            <h3>Robust Privacy and Do Not Track</h3>
+            <p>Thunderbird offers support for user privacy and remote image protection. To ensure a user's privacy, Thunderbird automatically blocks remote images in email messages.</p>
```

Comment:
Nit: We should use the `’` character consistently instead of switching between it and the single quote so, `user’s privacy...` instead of `user's privacy...`